tests/conversion_script/test_insights.py

```diff
@@ -16,9 +16,6 @@ def test_update_insights_inventory_non_success():
     with patch(
         "scripts.conversion_script.run_subprocess", return_value=(b"output", 1)
     ) as mock_popen:
-        with pytest.raises(
-            ProcessError, match="insights-client execution exited with code '1'"
-        ):
-            update_insights_inventory()
+        update_insights_inventory()
```

Comment:
With my above suggestion (https://github.com/oamg/convert2rhel-worker-scripts/pull/52/files#r1415917842) this can be reverted.